index.js

```diff
@@ -1,4 +1,20 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const { productsRouter, salesRouter } = require('./controllers');
+
+const app = express();
+
+const PORT = 3000;
```

Comment:
O que acha de definir a porta como uma variável de ambiente? Acho que ajuda a proteger informações sensíveis da aplicação. e sempre existe a possibilidade de configurar um _fallback_ caso a variável de ambiente não possa ser lida.
Outra vantagem é que, no caso de deploy, você não terá que alterar seu código.

Penso em mais ou menos o seguinte:

```suggestion
const PORT = process.env.PORT || 3000;
```